car-pool/README.md

```diff
@@ -29,7 +29,7 @@ We propose developing a transport agnostic CAR-based synchronization mechanism,
 
 This proposal resolves multiple major, persistent challenges that numerous production IPFS operators have experienced. We believe the scope of this work to be an important stepping stone in the maturation of IPFS and Filecoin in production settings.
 
-We propose extending the existing efforts to move specifically CAR files over HTTP (e.g. Filecoin point-to-point CAR, Qri’s DSync), and enable other transports such as WSS, HLS, and WebRTC. We know of several other projects in the broader ecosystem that would like to make use of this protocol, from storage providers and CDNs to decentralized social media protocols.
+We propose extending the existing efforts to move specifically CAR files over HTTP (e.g. Filecoin point-to-point CAR, Qri’s DSync), and enable other transports such as [WSS](https://datatracker.ietf.org/doc/html/rfc6455), [HTTP Streaming](https://datatracker.ietf.org/doc/html/rfc2616#section-3.6.1), and [WebRTC](https://datatracker.ietf.org/doc/html/rfc8835). We know of several other projects in the broader ecosystem that would like to make use of this protocol, from storage providers and CDNs to decentralized social media protocols.
```

Comment:
I didn't need to add the other links, but it felt weird not to after doing so for HTTP Streaming.  I chose to keep just HTTP Streaming vs adding verbiage around how we really mean chunked transfer-encoding.

Reply:
Links never hurt! Thanks 😄